llvm/test/Instrumentation/AddressSanitizer/module-flags-aarch64.ll

```diff
@@ -0,0 +1,31 @@
+; RUN: opt < %s -passes=asan -S | FileCheck %s
```

Comment:
Please could you add a comment saying what these tests are testing.

Reply:
ack.